psychopy/data.py

```diff
@@ -3464,8 +3464,8 @@ class QuestHandler(StairHandler):
         # create staircase object
         # trying to find out the point where subject's response is 50 / 50
         # if wanted to do a 2AFC then the defaults for pThreshold and gamma
-        # are good
-        staircase = data.QuestHandler(staircase._nextIntensity, 0.2,
+        # are good. As start value, we'll use 50% contrast, with SD = 20%
+        staircase = data.QuestHandler(0.5, 0.2,
```

Comment:
Aha, OK! I think you're right on startVal not being the starting point. Maybe it should be renamed prior in the function argument, or tGuess like in the actual quest code. Which is I think what they call it in the matlab QUEST that comes with the psychtoolbox, so that would be nice too!